### PR TITLE
Add `rebuild()` method for panel update.

### PR DESCRIPTION
**Pull request by jongmoon**
## Issue
#239 

## Details
Add `rebuild()` method for panel update.

### CODE REVIEW

File: src/Flicking.js [hunk 1]
```diff
@@ -384,6 +390,13 @@ export default class Flicking extends Mixin(Component).with(eventHandler) {
 		);
 	}
 
+	_clonePanel(v) {
+		const clone = v.cloneNode(true);
+
+		clone.classList.add(`${this.options.prefix}-clone`);
```
Comment (WoodNeck) [inline src/Flicking.js]: I think `utils.classList` is more preferred for browser compatibility.

File: test/unit/methods.spec.js [hunk 2]
```diff
@@ -109,7 +109,7 @@ describe("Methods call", function() {
 	describe("getPrevElement()", function() {
 		tutils.hooks.run();
 
-		// Given
+// Given
```
Comment (WoodNeck) [inline test/unit/methods.spec.js]: blank spaces are removed

File: src/Flicking.js [hunk 3]
```diff
@@ -1743,6 +1756,38 @@ export default class Flicking extends Mixin(Component).with(eventHandler) {
...
+
+		// panels' css values
+		this._initOriginalPanelStyle(panel.$list);
+
+		// Add clones
+		if (this._addClonePanels()) {
+			panel.$list = utils.toArray(this.$container.children);
+		}
+
+		this._setDefaultPanel(options.defaultIndex);
```
Comment (daybrush) [inline src/Flicking.js]: @jongmoon 
Shouldn't `defaultIndex` be the current index when calling `rebuild`?

File: src/Flicking.js [hunk 4]
```diff
@@ -1743,6 +1756,38 @@ export default class Flicking extends Mixin(Component).with(eventHandler) {
...
+	 *
+	 * @return {eg.Flicking} An instance of a module itself <ko>모듈 자신의 인스턴스</ko>
+	 */
+	rebuild() {
+		const panel = this._conf.panel;
+		const options = this.options;
+		const prefix = options.prefix;
+
+		// filter original panel (remove clones)
+		utils.toArray(this.$wrapper.querySelectorAll(`.${prefix}-clone`)).forEach(el => el.remove());
```
Comment (daybrush) [inline src/Flicking.js]: @jongmoon 

IE does not support 'remove' methods.

I think you should use parentNode.removeChild.

File: src/Flicking.js (hunk 1 above)
Comment (jongmoon) [reply src/Flicking.js]: Thank you!

File: src/Flicking.js (hunk 3 above)
Comment (jongmoon) [reply src/Flicking.js]: @daybrush I think `rebuild` is used for working like initial time. Maintaining current index is policy issue. How about to add parameter?

```js
flicking.rebuild({defaultIndex: 1});
```

File: src/Flicking.js (hunk 4 above)
Comment (jongmoon) [reply src/Flicking.js]: Awesome!

File: src/Flicking.js (hunk 3 above)
Comment (daybrush) [reply src/Flicking.js]: @jongmoon 

When I heard, I knew that the contents of the panel were updated in the `circular`.

Comment (daybrush) [reply src/Flicking.js]: @jongmoon

I think adding parameters is good.

**Pull request merged**